lib/python/qmk/commands.py

```diff
@@ -69,14 +69,13 @@ def parse_configurator_json(configurator_file):
 def run(command, *args, **kwargs):
     """Run a command with subprocess.run
     """
-    OS = platform.platform().lower()  # noqa (N806), uppercase name is ok in this instance
-
     if isinstance(command, str):
-        raise TypeError('command must be a sequence of items such as list or tuple.')
+        raise TypeError('`command` must be a sequence such as list or tuple.')
+
+    platform_id = platform.platform().lower()
 
-    if 'windows' in OS:
+    if 'windows' in platform_id:
         command = map(shlex.quote, command)
```

Comment:
```suggestion
        command = ' '.join(map(shlex.quote, command))
```
I think command needs to be a string in this context.

Reply:
Yeah, I get `TypeError: expected str, bytes or os.PathLike object, not map` without this.